samples/net/ocpp/README.rst

```diff
@@ -0,0 +1,65 @@
+.. zephyr:code-sample:: OCPP charge point
+   :name: OCPP charge point
+   :relevant-api: ocpp_api
+
+   Implement a OCPP charge point that connects to a Central System server and
+   simulates the meter readings.
+
+Overview
+********
+
+Open Charge Point Protocol (OCPP) is an application protocol for communication
+between Charge Points (Electric vehicle (EV) charging stations) and a central
+management system, also known as a charging station network.
+
+This ocpp sample application for Zephyr implements the ocpp library
+and establishes a connection to an Central System server using the web socket
+
+The source code for this sample application can be found at:
+:zephyr_file:`samples/net/ocpp`.
+
+Requirements
+************
+
+- Linux machine
+- STEM32 Discovery kit (32F769IDISCOVERY)
```

Comment:
Is this really hard requirement, or are we just depend on a board that has Ethernet?

Reply:
No, but tested on this board. Updated doc